bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/IDEWorkbenchMessages.java

```diff
@@ -9,23 +9,24 @@
  * SPDX-License-Identifier: EPL-2.0
```

Comment:
Would be nice to not do unnecessary formatting changes, at least not in combination with actual code changes that matter.

If this is done by the formatter, you can "undo" it with Ctrl+Z (or whichever undo hotkey you use) after a save.

Reply:
Thank you for the feedback. I wasn't sure about this change and whether everything on a PR is still supposed to go into one single commit. I considered the formatting of the contributors list broken and aimed to fix it so that it would match the other classes I'd seen.

I reverted that change now, so that the only change in the list is the new entry in the list.